bindings/python/config_validator.h

```diff
@@ -0,0 +1,66 @@
+/*
+//@HEADER
+// *****************************************************************************
+//
+//                               config_validator.h
+//             DARMA/vt-tv => Virtual Transport -- Task Visualizer
+//
+// Copyright 2019-2024 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS). Under the terms of Contract DE-NA0003525 with NTESS, the U.S.
+// Government retains certain rights in this software.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//
+// * Redistributions of source code must retain the above copyright notice,
+//   this list of conditions and the following disclaimer.
+//
+// * Redistributions in binary form must reproduce the above copyright notice,
+//   this list of conditions and the following disclaimer in the documentation
+//   and/or other materials provided with the distribution.
+//
+// * Neither the name of the copyright holder nor the names of its
+//   contributors may be used to endorse or promote products derived from this
+//   software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+// LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+// POSSIBILITY OF SUCH DAMAGE.
+//
+// Questions? Contact darma@sandia.gov
+//
+// *****************************************************************************
+//@HEADER
+*/
+// A2DD.h
+#ifndef vt_tv_config_validator_h
+#define vt_tv_config_validator_h
+
+#include <yaml-cpp/yaml.h>
+
+namespace vt::tv::bindings::python {
+    /**
+     * ConfiValidator Class
+     */
+    class ConfigValidator
+    {
+        public:
+            std::array<std::string, 2> requiredParameters = {"output_visualization_dir", "output_visualization_file_stem"};
+            YAML::Node config;
+            bool isValid();
+            std::string getMissingRequiredParameters();
+            ConfigValidator(YAML::Node configData) {
```

Comment:
For consistency I think (as e.g. in the `ObjectInfo` class)
- it is better to use the `in_` parameter prefix in constructor `ConfigValidator(YAML::Node in_config)` (and name the attribute without the `in_` prefix)
- use the parameter initialization feature as possible
- use private attribute as possible (with getter if needed)
```c++
ConfigValidator(YAML::Node in_config) 
  :config(in_config) {}
```